samples/v1.3/Scenarios/ExpenseReport.json

```diff
@@ -0,0 +1,824 @@
+{
```

Comment:
This uses the old ExpenseReport.json which has an accessibility bug. Can you update to use the latest one from main?